src/mode/modeHandler.ts

```diff
@@ -185,41 +198,41 @@ export class SearchState {
   public getNextSearchMatchPosition(startPosition: Position, direction = 1): { pos: Position, match: boolean} {
     this._recalculateSearchRanges();
 
-    if (this.matchRanges.length === 0) {
+    if (this._matchRanges.length === 0) {
       // TODO(bell)
       return { pos: startPosition, match: false };
     }
 
-    const effectiveDirection = direction * this.searchDirection;
+    const effectiveDirection = direction * this._searchDirection;
 
-    if (effectiveDirection === 1) {
-      for (let matchRange of this.matchRanges) {
+    if (effectiveDirection === SearchDirection.Forward) {
+      for (let matchRange of this._matchRanges) {
         if (matchRange.start.compareTo(startPosition) > 0) {
           return { pos: Position.FromVSCodePosition(matchRange.start), match: true };
         }
       }
 
       // Wrap around
       // TODO(bell)
-      return { pos: Position.FromVSCodePosition(this.matchRanges[0].start), match: true };
+      return { pos: Position.FromVSCodePosition(this._matchRanges[0].start), match: true };
     } else {
-      for (let matchRange of this.matchRanges.slice(0).reverse()) {
+      for (let matchRange of this._matchRanges.slice(0).reverse()) {
         if (matchRange.start.compareTo(startPosition) < 0) {
           return { pos: Position.FromVSCodePosition(matchRange.start), match: true };
         }
       }
 
       // TODO(bell)
```

Comment:
This means that Vim rings a bell here (or flashes the screen or otherwise indicates an error has occurred).